python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputSchemaRenderer.tsx

```diff
@@ -42,16 +42,18 @@ function SchemaRenderer({ input, schema, onChangeInput }: SchemaRendererProps) {
 
   return (
     <Flex direction="column">
-      <DataRenderer
-        schema={dataSchema}
-        data={data}
-        onChangeData={onChangeData}
-      />
+      {dataSchema && (
+        <DataRenderer
```

Comment:
Why do we need another `DataRenderer` class instead of re-using `AttachmentsRenderer` for audio files?

Reply:
DataRenderer is for the data key. Some prompts, like GPT-4V will have both data (string prompt) and attachments (images)